.github/CODEOWNERS

```diff
@@ -71,6 +75,11 @@
 
 # Modules
 
+/modules/                                     @godotengine/_engine
+/modules/**/doc_classes/                      @godotengine/_engine @godotengine/documentation
+/modules/**/icons/                            @godotengine/_engine @godotengine/usability
+/modules/**/tests/                            @godotengine/_engine @godotengine/tests
```

Comment:
Do we want to add the various modules specific `editor` sub folders to something? We use `@godotengine/_editor` for `/editor/` for with these changes

Reply:
Ohh, good point! Yeah, I'll go ahead and add those

If we wanted to be *really* picky, we could give the same treatment to SCSub/.py files, but that'd probably make the file too bloated. There's an argument to be made of somehow generating the `CODEOWNERS` file from a more readable baseline, but that's beyond the scope of what I wanted to achieve in this PR